src/summary/TabularListVisualizer.jsx

```diff
@@ -251,6 +251,10 @@ export default class TabularListVisualizer extends Component {
             const isInsertable = Lang.isUndefined(element.isInsertable) ? true : element.isInsertable;
             const isUnsigned = element.isUnsigned || false;
             let elementText = element.value;
+            if (arrayIndex === 3) {
```

Comment:
[GQ1] We can't change TabularListVisualizer to have code specific to any table. Incorporating asNeeded is specific to medications and tabular list visualizer should stay generic. I can explain better in person maybe...